packages/docusaurus-theme-common/src/translations/blogTranslations.tsx

```diff
@@ -77,3 +77,13 @@ export function BlogAuthorsListViewAllLabel(): ReactNode {
     </Translate>
   );
 }
+
+export function BlogAuthorNoPostLabel(): ReactNode {
```

Comment:
```suggestion
export function BlogAuthorNoPostsLabel(): ReactNode {
```

Let's be consistent with the id ending with an s